src/prompts.ts

```diff
@@ -111,6 +111,26 @@ const getOneLineCommitInstruction = () =>
     ? 'Craft a concise commit message that encapsulates all changes made, with an emphasis on the primary updates. If the modifications share a common theme or scope, mention it succinctly; otherwise, leave the scope out to maintain focus. The goal is to provide a clear and unified overview of the changes in a one single message, without diverging into a list of commit per file change.'
     : '';
 
+/**
+ * Get the context of the user input
+ * @param extraArgs - The arguments passed to the command line
+ * @example
+ *  $ oco -- This is a context used to generate the commit message
+ * @returns - The context of the user input
+ */
+const userInputCodeContext = () => {
+  const args = process.argv;
```

Comment:
i think this function should accept an argument `context: string`, wdyt?

Reply:
To implement this in a more effective manner, I switched to a flag called context. This approach makes the code more declarative and self-explanatory. Users can learn about this feature by running the oco --help command, which provides detailed information on how to use the context flag and its purpose within the application

Reply:
Check again when you can